Cargo.toml

```diff
@@ -21,6 +21,15 @@ include = [
 
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
+[lib]
+name = "suckit"
+path = "src/lib.rs"
+
+[[bin]]
+name = "suckit"
+path = "src/bin/suckit.rs"
```

Comment:
I think I'd rather have this file directly in the `src` directory as bin/ makes it look like a build file.
This is what [clippy does](https://github.com/rust-lang/rust-clippy/blob/28e769951fd6ed98c625b4f2d9bab046034ca2f2/Cargo.toml#L15)

Reply:
@CohenArthur bin directory will have other files related to binaries e.g. other binaries or even yaml for clap.

common pattern for this reason is not to mix them up and this was as low touch to ceate the separation with minimal touch so they live in different directories for clarity.

Reply:
Gotcha, thanks!

Reply:
thanks for clarifying, I need to explain my PRs better